pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,280 @@
+    the setpoint, reading back the current position,
+    actuating the motion, and stopping the motion.
+
+    Attributes:
+        setpoint (EpicsSignal): An EPICS signal for setting the desired
+        position of the axis.
+        readback (EpicsSignal): An EPICS signal for reading back the current
+        position of the axis.
+        actuate (EpicsSignal): An EPICS signal for initiating the motion of
+        the axis.
+        actuate_value (any, optional): The value to be set on the 'actuate'
+        signal to initiate motion (default is 1).
+        stop_signal (EpicsSignal): An EPICS signal for stopping the motion of
+        the axis.
+        stop_value (any, optional): The value to be set on the 'stop_signal'
+        to stop motion (default is 1).
+
+    Args:
+        prefix (str): The EPICS PV prefix for this axis.
+        axis (str): The axis identifier ('X', 'Y', 'Z', 'rX', 'rY', 'rZ').
+        sync_setpoints (callable or None): A callback function to synchronize
+            setpoints before moving the axis (default is None).
+        **kwargs: Additional keyword arguments to pass to the base class
+        constructor.
+
+    Methods:
+        move(position,
+             wait=True,
+             timeout=None,
+             moved_cb=None,
+             sync_enable=True
+             ):
+            Move the axis to the specified position.
+
+    Example:
+        >>> axis_x = SQR1Axis(prefix="SQR1:AXIS_X", axis="X")
+        >>> axis_x.move(5.0)  # Move the X-axis to position 5.0
+    """
+
+    setpoint = FCpt(EpicsSignal, "{prefix}:TARGET:{_axis}", kind="normal")
+    readback = FCpt(EpicsSignal, "{prefix}:TARGET:{_axis}:RBV", kind="hinted")
+    actuate = Cpt(EpicsSignal, ":MOV", kind="normal")
+    actuate_value = 1
+    stop_signal = Cpt(EpicsSignal, ":KILL", kind="normal")
+    stop_value = 1
+
+    def __init__(
+        self,
+        prefix: str,
+        axis: Axis,
+        sync_setpoints: typing.Callable | None = None,
+        **kwargs,
+    ):
+        self._axis = axis.value
+        self._sync_setpoints = sync_setpoints
+        super().__init__(prefix, **kwargs)
+
+    def move(
+        self,
+        position: float,
+        wait: bool = True,
+        timeout: float = None,
+        moved_cb: typing.Callable | None = None,
+        sync_enable: bool = True,
+    ):
+        if sync_enable and self._sync_setpoints:
+            self._sync_setpoints()
+        return super().move(position, wait, timeout, moved_cb)
+
+
+class SQR1(Device):
+    """
+    This class represents the entire square-one tri-sphere motion system,
+    which consists of multiple axes (X, Y, Z, rX, rY, rZ) that can be
+    controlled individually. It aggregates instances of the `SQR1Axis` class
+    for each axis and provides methods for multi-axis movement and
+    stopping the motion.
+
+    Attributes:
+        x (SQR1Axis): An instance of the `SQR1Axis` class representing
+        the X-axis.
```

Comment:
Each of these is rendering kind of strangely in the html docs:
![image](https://github.com/pcdshub/pcdsdevices/assets/10647860/57bd277b-7eda-49d1-a082-33f955c8f227)

Reply:
I'm not sure the right way to reformat this because I've only used the numpy style docstrings
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#google-vs-numpy

Reply:
I think the problem with second line indentation